src/lab_sim/objectives/plan_and_save_trajectory.xml

```diff
@@ -39,7 +40,7 @@
       />
       <Action ID="PlanMTCTask" solution="{mtc_solution}" task="{mtc_task}" />
       <Action ID="WaitForUserTrajectoryApproval" />
-      <!-- The following behavior can be found in the example_behaviors package. Remove the COLCON_IGNORE file and `moveit_pro build user_workspace` to use. -->
+      <!--The following behavior can be found in the example_behaviors package.-->
```

Comment:
small nit, this is referring to the source code right? Maybe it should read "The following behavior's code can be found in the `example_behaviors` package"

Reply:
not relevant for this merge